src/NzbDrone.Core/Notifications/Webhook/WebhookService.cs

```diff
@@ -0,0 +1,124 @@
+﻿using FluentValidation.Results;
+using NLog;
+using NzbDrone.Common.Processes;
+using NzbDrone.Core.MediaFiles;
+using NzbDrone.Core.Tv;
+using NzbDrone.Core.Validation;
+using NzbDrone.Core.Rest;
+using RestSharp;
+using NzbDrone.Core.Qualities;
+using NzbDrone.Core.Parser.Model;
+using System.Collections.Generic;
+
+namespace NzbDrone.Core.Notifications.Webhook
+{
+    public interface IWebhookService
+    {
+        void OnDownload(Series series, EpisodeFile episodeFile, WebhookSettings settings);
+        void OnRename(Series series, WebhookSettings settings);
+        void OnGrab(Series series, RemoteEpisode episode, QualityModel quality, WebhookSettings settings);
+        ValidationFailure Test(WebhookSettings settings);
+    }
+
+    public class WebhookService : IWebhookService
+    {
+        public WebhookService(IProcessProvider processProvider, Logger logger)
```

Comment:
you don't actually need the constructor anymore since it doesn't actually do anything. You can just kill it.